sys-boot/raspberrypi-firmware/raspberrypi-firmware-1.20190215.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mount-boot readme.gentoo-r1
+
+DESCRIPTION="Raspberry Pi (all versions) bootloader and GPU firmware"
+HOMEPAGE="https://github.com/raspberrypi/firmware"
+SRC_URI="https://github.com/raspberrypi/firmware/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 BSD"
```

Comment:
This is not correct. Old LICENSE value was correct.
Everything but GPL stuff is exactly under a license known in Gentoo tree as raspberrypi-videocore-bin.